src/physicsCalculator/Calculator.h

```diff
@@ -1,52 +1,92 @@
 /**
  * @file
  *
- * @brief Defines a default interface for the physics caculators
+ * @brief Defines a default interface for the physics calculators.
  */
 
 #pragma once
 
-class ParticleContainer;
-class Environment;
-
+#include "../Environment.h"
+#include "../ParticleContainer.h"
```

Comment:
you shouldn't need to provide exact path, this should be handled in CMake file. I am refering to `../` here